orttraining/orttraining/training_ops/cuda/nn/dropout_impl.cu

```diff
@@ -41,12 +41,16 @@ void DropoutGradientKernelImpl(
     const T* dY_data,
     const bool* mask_data,
     const float ratio,
+    const bool training_mode,
     T* dX_data) {
-  if (ratio == 0.0f) {
+  if (ratio == 0.0f or !training_mode) {
```

Comment:
nit: i find "or" kind of unexpected, consider "||" for consistency with most of the other code

Reply:
This is me coding python on a c++ file :P